databuilder/publisher/neo4j_csv_publisher.py

```diff
@@ -371,29 +369,17 @@ def _create_props_body(self,
         :param record_dict: A dict represents CSV row
         :param excludes: set of excluded columns that does not need to be in properties (e.g: KEY, LABEL ...)
         :param identifier: identifier that will be used in CYPHER query as shown on above example
-        :param is_update: Creates property body for update statement in MERGE
         :return: Properties body for Cypher statement
         """
-        template_params = {}
         props = []
         for k, v in record_dict.items():
             if k in excludes:
-                template_params[k] = v
                 continue
 
-            # if isinstance(str, v):
-            # escape backslash for Cypher query
-            v = v.replace('\\', '\\\\')
-            # escape quote for Cypher query
-            v = v.replace('\'', "\\'")
-
             if k.endswith(UNQUOTED_SUFFIX):
                 k = k[:-len(UNQUOTED_SUFFIX)]
-                props.append('{id}.{key} = {val}'.format(id=identifier, key=k, val=v))
-            else:
-                props.append("""{id}.{key} = '{val}'""".format(id=identifier, key=k, val=v))
 
-            template_params[k] = v
+            props.append('{id}.{key} = {val}'.format(id=identifier, key=k, val=f'${k}'))
```

Comment:
The property value is taken out wherein a param token (i.e. `$param`) is put in.